echopype/convert/utils/ek_raw_io.py

```diff
@@ -6,18 +6,18 @@
 Contains low-level functions called by ./ek_raw_parsers.py
 """
 
-import logging
 import struct
 from io import SEEK_CUR, SEEK_END, SEEK_SET, BufferedReader, FileIO
 
 import fsspec
 from fsspec.implementations.local import LocalFileSystem
 
+from ...utils.log import _init_logger
 from . import ek_raw_parsers as parsers
 
 __all__ = ["RawSimradFile"]
 
-log = logging.getLogger(__name__)
+log = _init_logger(__name__)
```

Comment:
Change the name of the assigned variable to `logger`, just for consistency with usage elsewhere.
```suggestion
logger = _init_logger(__name__)
```